src/components/search/Search.js

```diff
@@ -89,13 +190,16 @@ const Search = (props) => {
               } = suggestion
 
               // Allow handleSelect to access the place id (see above)
-              descriptionToPlaceId.current[description] = place_id
+              if (place_id) {
+                descriptionToPlaceId.current[description] = place_id
+              }
```

Comment:
You don't need these if clause anymore right? You can move line 194 outside and remove the if?